vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/confirmdialog/tests/FeaturesDiy.java

```diff
@@ -4,9 +4,7 @@
 import com.vaadin.flow.component.button.Button;
 import com.vaadin.flow.component.confirmdialog.ConfirmDialog;
 import com.vaadin.flow.component.html.H2;
-import com.vaadin.flow.component.html.Label;
 import com.vaadin.flow.component.html.Span;
-import com.vaadin.flow.component.icon.VaadinIcon;
```

Comment:
Aaand now the build doesn't pass because you removed the VaadinIcon import

Reply:
🤬 The bloody project don't build properly anymore because time has passed (and probably my snapshots our out of sync). 

I have  a suggestion for you:

 * stop nitting, at least for PRs made by non-team members and PRs that have been hanging in the queue for more than a day or two
 * if there are tiny details you want to change in this kind of PRs, just change them. There should be no need for anybody to review those if you work with the codebase daily. 

This saves your time, my time and @knoobie's time.

Reply:
I understand both sides here 😬 Just an idea from an outsider, that I've stolen from the spring team: they polish external contribution after the merge, making the process for both parties way more attractive and faster 😇

Reply:
As I already said in https://github.com/vaadin/flow-components/pull/5495#discussion_r1345854188, I tried to push the fixes to your branch (regarding the "nitty" things) but since you don't allow maintainers to edit, I have no other choice than asking you to fix them.

Reply:
That is a good one @knoobie 👍 Based on what I discussed with mr Spring Jürgen, they also don't have that strict core review processes, but do it when they see it appropriate. And then Jürgen checks all through before the release still. I think a lot more maintenance tasks could be done if we'd rationalize our processes on trivial tasks.

And BTW. My checkbox is on, we learn by doing ;-)